unattended_installer/builder.sh

```diff
@@ -265,4 +268,26 @@ function builder_main() {
     fi
 }
 
+function checkFilebeatURL() {
+
+    # Import variables
+    . ${resources_installer}/installVariables.sh
+    new_filebeat_url="https://raw.githubusercontent.com/wazuh/wazuh/master/extensions/elasticsearch/7.x/wazuh-template.json"
+
+    # Get the response of the URL and check it
+    response=$(curl --write-out '%{http_code}' --silent --output /dev/null $filebeat_wazuh_template)
+    if [ $response != "200" ]; then
+       	response=$(curl --write-out '%{http_code}' --silent --output /dev/null $new_filebeat_url)
+
+        # Display error if both URLs do not get the resource
+        if [ $response != "200" ]; then
```

Comment:
Use double quotes in variables. See https://bertvv.github.io/cheat-sheets/Bash.html

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/2001/commits/7e37338197a551ead63bb937339751420e0b2ef5